notebooks/uvis_pam_corrections/WFC3 UVIS Pixel Area Map Corrections.ipynb

```diff
@@ -0,0 +1,733 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, let's retrieve our data and headers using `astropy.io.fits()`. \n",
+    "We start by opening the FLT file and reviewing the data structure:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "file_loc_512='mastDownload/HST/icjd05saq/icjd05saq_flt.fits'\n",
+    "hdu_512=fits.open(file_loc_512)\n",
+    "hdu_512.info()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We need the science data and the science header. We view the data and print the first 10 lines of each to peek at their contents:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_512=hdu_512[1].data\n",
+    "\n",
+    "#We use zcale to find the min and max for plotting\n",
+    "vmin_512, vmax_512 = zscale(data_512)\n",
+    "\n",
+    "plt.imshow(data_512, vmin=vmin_512, vmax=vmax_512, origin='lower')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "scihdr_512=hdu_512[1].header\n",
+    "scihdr_512[0:10]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, to define the subarray coordinates that we will 'cut' out of the Pixel Area Map, we look at certain keywords.\\\n",
+    "Namely, we look at:\n",
+    "- scihdr['LTV1']: 'offset in X to subsection start' \n",
+    "\n",
+    "- scihdr['LTV2']: 'offset in Y to subsection start'\n",
+    "\n",
+    "- scihdr['NAXIS1']: 'length of data axis 1'\n",
+    "\n",
+    "- scihdr['NAXIS2']: 'length of data axis 2'\n",
+    "\n",
+    "These define the x and y pixel coordinates of the four corners of the science image."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "x0 = int(np.abs(scihdr_512['LTV1']))\n",
+    "y0 = int(np.abs(scihdr_512['LTV2']))\n",
+    "x1 = int(x0 + scihdr_512['NAXIS1'])\n",
+    "y1 = int(y0 + scihdr_512['NAXIS2'])\n",
+    "\n",
+    "print (f'(x0, y0, x1, y1) = ({x0}, {y0}, {x1}, {y1})')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4. Downloading the Correct Pixel Area Map<a id=\"pam\"></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4.1 Download PAM Files from the WFC3 Website <a id=\"PAMload\"></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The UVIS CCD has two chips: UVIS1 and UVIS2.\n",
+    "\n",
+    "First, please go to https://www.stsci.edu/hst/instrumentation/wfc3/data-analysis/pixel-area-maps and download the UVIS1 and UVIS2 PAMs under the \"Download Pixel Area Maps\" header. "
```

Comment:
First, please go to [the PAM website](https://www.stsci.edu/hst/instrumentation/wfc3/data-analysis/pixel-area-maps) and ...